Packages/vcs/Lib/VTKPlots.py

```diff
@@ -1184,7 +1171,22 @@ def plotContinents(self,x1,x2,y1,y2,projection,wrap,tmpl):
       return {}
 
   def renderTemplate(self,tmpl,data,gm,taxis,zaxis):
-    tmpl.plot(self.canvas,data,gm,bg=self.bg)
+    ## ok first basic template stuff, let's store the displays
+    ## because we need to return actors for min/max/mean
+    displays = tmpl.plot(self.canvas,data,gm,bg=self.bg)
+    returned = {}
+    for d in displays:
+        if d is None:
+          continue
+        texts=d.backend.get("vtk_backend_text_actors",[])
+        for t in texts:
+          ## ok we had a text actor, let's see if it's min/max/mean
+          txt = t.GetInput()
+          s0=txt.split()[0]
+          if s0 in ["Min","Max","Mean"]:
+              returned["vtk_backend_%s_text_actor" % s0] = t
+              self.canvas.display_names.remove(d.name)
```

Comment:
it has to be here, render template created these display obejct so that they would be added to the returned, but that throws things off for @chaosphere2112 because now after the "plot" call we have 4 dispaly objects rather than one so they need to be removed. @chaosphere2112 is going to look into this as well and decides if he needs to return (and delete) more.